release_notes/ocp_3_10_release_notes.adoc

```diff
@@ -54,33 +54,550 @@ documentation.
+
+Persistent volume (PV) resize is currently in
+xref:ocp-310-technology-preview[Technology Preview] and not for production
+workloads.
+
+You can expand persistent volume claims online from {product-title} for CNS
+glusterFS, Cinder, and GCE PD.
+
+. Create a storage class with `allowVolumeExpansion=true`.
+. The PVC uses the storage class and submits a claim.
+. The PVC specifies a new increased size.
+. The underlying PV is resized.
+
+[[ocp-310-ceph-provisioner]]
+==== Ceph Provisioner
+
+WIP
+
+[[ocp-310-CSI]]
+==== Container Storage Interface (Technology Preview)
+
+Container Storage Interface (CSI) is currently in
+xref:ocp-310-technology-preview[Technology Preview] and not for production
+workloads.
+
+CSI allows {product-title} to consume storage from storage backends that
+implement the link:https://github.com/container-storage-interface/spec[CSI
+interface] as
+xref:../architecture/additional_concepts/storage.adoc#architecture-additional-concepts-storage[persistent
+storage].
+
+See
+xref:../install_config/persistent_storage/persistent_storage_csi.adoc#install-config-persistent-storage-persistent-storage-csi[Persistent
+Storage Using Container Storage Interface (CSI)] for more information.
+
+[[ocp-310-Local-Storage]]
+==== Local Storage
+WIP
+
+[[ocp-310-tenant-driven-storage-snapshotting]]
+==== Tenant-driven Storage Snapshotting (Technology Preview)
+
+Tenant-driven storage snapshotting is currently in
+xref:ocp-310-technology-preview[Technology Preview] and not for production
+workloads.
+
+Tenants now have the ability to leverage the underlying storage technology
+backing the persistent volume (PV) assigned to them to make a snapshot of their
+application data. Tenants can also now restore a given snapshot from the past to
+their current application.
+
+An external provisioner is used to access the EBS, GCE pDisk, and HostPath, and
+Cinder snapshotting API. This Technology Preview feature has tested EBS and
+HostPath. The tenant must stop the pods and start them manually.
+
+. The administrator runs an external provisioner for the cluster. These are images
+from the Red hat Container Catalog.
+
+. The tenant made a PVC and owns a PV from one of the supported storage
+solutions.The administrator must create a new `StorageClass` in the cluster with:
++
+----
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: snapshot-promoter
+provisioner: volumesnapshot.external-storage.k8s.io/snapshot-promoter
+----
+
+. The tenant can create a snapshot of a PVC named `gce-pvc` and the resulting
+snapshot will be called `snapshot-demo`.
++
+----
+$ oc create -f snapshot.yaml
+
+apiVersion: volumesnapshot.external-storage.k8s.io/v1
+kind: VolumeSnapshot
+metadata:
+  name: snapshot-demo
+  namespace: myns
+spec:
+  persistentVolumeClaimName: gce-pvc
+----
+
+. Now, they can restore their pod to that snapshot.
++
+----
+$ oc create -f restore.yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: snapshot-demo
```

Comment:
Better use a different name with the snapshot name as it can confuse end user.

Reply:
@liangxia I was previously instructed by @sferich888 to change it to match. Eric, can I change it back?